### PR TITLE
Added support to create video thumbnails from mp4 derivatives. DCMKEY-576

### PR DESCRIPTION
**Pull request by lsitu**
https://lib-jira.ucsd.edu:8443/browse/DCMKEY-576

### CODE REVIEW

**Comment by mcritchlow** [comment]
is work on https://lib-jira.ucsd.edu:8443/browse/DAMSKEY-565 a dependency for this ticket? or do you think this feature is safe while we work on that issue separately?

**Comment by lsitu** [comment]
@mcritchlow I think it could be kind of dependency but we need to figure what tool we can use for https://lib-jira.ucsd.edu:8443/browse/DAMSKEY-565 first. While we work on DAMSKEY-565, this feature should be safe until the new tool and implementation is in place.

File: WebContent/WEB-INF/jsp/controlPanel.jsp [hunk 1]
```diff
@@ -146,6 +146,9 @@
 									   <span class="submenuText"><input type="radio" name="size" value="v" > MP3/MP4 </span><br />
 									</fieldset>
 						</div>
+						<div title="Choose the frame no. for video thumbnail creation. Default is 100." class="specialmenuText">
```
Comment (mcritchlow) [inline WebContent/WEB-INF/jsp/controlPanel.jsp]: was this option to choose a video frame asked for? i'm not seeing it in the ticket description. apologies if I missed it.

File: src/edu/ucsd/library/xdre/collection/DerivativeHandler.java [hunk 2]
```diff
@@ -106,60 +107,114 @@ public boolean execute() throws Exception{
...
+
+	        		// handle video and audio derivatives creation
+	        		if (sizesList.indexOf("v") >= 0) {
+
+	        			File dst = null;    // derivative file created
+	        			sizesList.remove(sizesList.indexOf("v"));
+	        			Map<String, String> params = new HashMap<String, String>();
+	        			boolean successful = false;
+	        			try {
+			        		if (isVideo(fileID, use)) {
```
Comment (mcritchlow) [inline src/edu/ucsd/library/xdre/collection/DerivativeHandler.java]: :+1: thanks for the refactor

File: WebContent/WEB-INF/jsp/controlPanel.jsp (hunk 1 above)
Comment (lsitu) [inline WebContent/WEB-INF/jsp/controlPanel.jsp]: The next line #L150 will do the job.

**Pull request merged**